priv/schemas/bundle_config_schema_v4.yaml

```diff
@@ -110,7 +110,9 @@ definitions:
         additionalProperties:
           type: string
       output:
-        type: string
+        type: object
```

Comment:
We should really remove `subcommands`. Now that we have git style subcommand support it shouldn't be something we support.